examples/logger.py

```diff
@@ -3,20 +3,51 @@
 """
 
 import time
+import logging
 from typing import Any, Optional, Tuple
-
 from PyQt5.QtCore import QObject, pyqtSlot, pyqtSignal
 from PyQt5.QtWidgets import QWidget, QVBoxLayout, QPushButton, QTextEdit, QLabel, QDialogButtonBox
-
 from qiwis import BaseApp
 
+
+class Signaller(QObject):
+    """Signal for LoggingHandler."""
+    signal = pyqtSignal(str)
+
+
+class LoggingHandler(logging.Handler):
+    """Handler for logger.
+
+    Sends a log message to connected function using emit.
+
+    Signals:
+        signal(log): A formatted log message is emitted.    
+    """
+
+    def __init__(self, slotfunc):
+        """Connect an input function to the signal.
```

Comment:
You should start this docstring with "Extended" as described in #44.
For example:
```python
"""Extended.

Connects an input function to the signal.

Args:
    slotfunc: A slot function connected to the signaller's signal.
"""
```

Reply:
In addition, please notice that "Connects" and "A slot ...".

Reply:
You should start a sentence with a capital letter.
Please notice the argument docstring of `slotfunc`.
I changed some expression.

FYI: We call the functions bounded by a class instances _methods_.

Reply:
Please read this @Aijuh

Reply:
I understand as add argument docstring for `slotfunc`.

Since slotfunc for loggingHandler is addLog method in LoggerApp, I add that part.